server/app/services/applicant/question/AddressQuestion.java

```diff
@@ -318,6 +320,16 @@ public ImmutableList<Path> getAllPaths() {
         getServiceAreaPath());
   }
 
+  @Override
+  public ImmutableMap<Path, String> getJsonEntries() {
+    return applicantQuestion.getContextualizedScalars().keySet().stream()
+        .filter(path -> !Scalar.getMetadataScalarKeys().contains(path.keyName()))
+        .collect(
+            toImmutableMap(
+                path -> path,
+                path -> applicantQuestion.getApplicantData().readAsString(path).orElse("")));
+  }
```

Comment:
**question (blocking):** the previous version of the code used `scalarAnswersInDefaultLocale` but this doesn't, why is that?

Reply:
Looking at my other comment on this file, there's no `case ADDRESS` so it goes to the default, `answerData.scalarAnswersInDefaultLocale()`. That AutoValue field is only set in `ReadOnlyApplicantProgramServiceImpl::getScalarAnswers`, where there also isn't a case for ADDRESS, and we fall back to the default `getContextualizedScalars()`.

To be honest I haven't fully understood the differences between "scalar answers" and "contextualized scalars" - I've just been moving around code and trying to keep things equivalent.

Reply:
"contextualized scalars" include paths that are "contextualized" i.e. they include array indices for parent enumerators, vs the uncontextualized path that has an empty array access (`[]`) in it

Reply:
Thanks for explaining. Resolving this given my explanation and because the diff in the description is clean for address. Unresolve if you have any more concerns.